staging/src/k8s.io/kubectl/pkg/cmd/attach/attach_test.go

```diff
@@ -349,6 +355,14 @@ func TestAttach(t *testing.T) {
 			if remoteAttach.url.Query().Get("container") != "bar" {
 				t.Errorf("%s: Did not have query parameters: %s", test.name, remoteAttach.url.Query())
 			}
+			if test.expectedErrOut != nil {
+				for _, expect := range test.expectedErrOut {
+					if !strings.Contains(errOut.String(), expect) {
+						t.Errorf("%s: Stderr did not contain expected message for attach request:\n expected:\n\t%s\n got:\n\t%s", test.name, expect, strings.ReplaceAll(errOut.String(), "\n", ""))
```

Comment:
nit: In my opinion, new lines hurt the code readability. What about this, or something like that?
```suggestion
t.Errorf("%s: expected message %s not found, got: %s", test.name, expect, strings.ReplaceAll(errOut.String(), "\n", ""))
```